lib/stat_tracker.rb

```diff
@@ -18,4 +18,9 @@ def self.from_csv(locations)
     StatTracker.new(games, teams, game_teams)
   end
 
+  def average_goals_per_game
+    total_goals = @games.sum{ |row| row[:away_goals] + row[:home_goals] }
+    (total_goals.to_f / @games.length).round(2)
```

Comment:
maybe change the variable row to game to keep our variables similar to each other throughout the methods

Reply:
great idea. done.